backend/editor/entries.py

```diff
@@ -452,13 +452,23 @@ async def update_node(self, label, entry, new_node):
         curr_node = result[0]["n"]
         deleted_keys = set(curr_node.keys()) - set(new_node.keys())
 
+        # Check for deleted keys having associated comments and delete them too
+        for key in deleted_keys:
+            comments_key = key + "_comments"
+            if new_node.get(comments_key) is not None:
```

Comment:
```suggestion
            if curr_node.get(comments_key) is not None:
```
Or am I mistaken?

Reply:
I think it's better to use `new_node`, because if there was no comments before (i.e in `curr_node`) but some comments were added to `new_node`, using `curr_node.get` would not add the comments to the deleted keys, and they would then be added to the node in neo4j.

Reply:
Yep, you are completely right!